azure-iot-device/azure/iot/device/common/pipeline/pipeline_stages_base.py

```diff
@@ -736,32 +731,146 @@ def do_retry():
 ]
 
 
+class ReconnectState(object):
```

Comment:
>ReconnectState [](start = 6, length = 14)

it feels these are getting used liked enums. though we can not use enums for some reasons that i have quite forgotten , may be it is  a good idea to use upper caps for all these variables

Reply:
I feel as though these probably don't really need to be in a class at all, although it's not a big deal either way

---
In reply to: [377910185](https://github.com/Azure/azure-iot-sdk-python/pull/460#discussion_r377910185) [](ancestors = 377910185)